home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/cooperation/invitation/cooperation/CooperationInvitationServiceImpl.java

```diff
@@ -44,9 +44,9 @@ protected InvitationDto saveInvitation(InvitationDto invitationDto) {
         var cooperationInvitation =
             mapper.convert(cooperationInvitationDto, CooperationInvitation.class);
 
-        if (isCooperationInvitationNonExists(invitationDto.getEmail(), cooperationInvitation.getCooperationName())) {
+        if (isCooperationInvitationNonExists(invitationDto.getEmail(), cooperationInvitation.getCooperationId())) {
```

Comment:
I know that it isn't your code but else is unreachable here or shouldn't be a client error.
Please change:
`throw new BadRequestHomeException("Invitation already exist for cooperation");`
to
`throw new IllegalStateException("Invitation already exist for cooperation");`

So it will be 500 instead of 400